apps/ett/e2e-ett/src/support/dashboard.po.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * @license
+ * Copyright 2021 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export const getTitle = () =>
+  cy.findByRole('heading', { level: 2, name: /dashboard/i });
```

Comment:
*Role* in Angular Testing Library and DOM Testing Library means *semantic role*/*accessible role*. It could be the sematic role of an `<h2>` element (`heading`, level 2) as in this case or any other native HTML element. It could be an HTML `role` attribute, or it could be and `aria-role` attribute.